export_to_godot_tileset.js

```diff
@@ -6,8 +6,9 @@ class GodotTilesetExporter {
         this.tileset = tileset;
         this.fileName = fileName;
         // noinspection JSUnresolvedFunction
-        this.projectRoot = getResPath(this.tileset.property("projectRoot"), fileName);
-        this.spriteImagePath = this.tileset.image.replace(this.projectRoot, "");
+        this.projectRoot = getResPath(this.tileset.property("projectRoot"), this.tileset.image);
+        this.filePath = this.tileset.image.substring(0, this.tileset.image.lastIndexOf("/"));
```

Comment:
```suggestion
        this.filePath = FileInfo.path(this.tileset.image);
```
But are we sure the file path should be based on the image, rather than `fileName`?

Reply:
I am positive for this one we should use the tileset.image.  This filepath is being used to update the image path for the tileset, not the tileset itself. Since the image can be in a different folder from the tileset file this can cause issues with the replacement logic.  

IE this is the output that filepath is being used for:
[ext_resource path="res://maps/terrain.png" type="Texture" id=1]

I do like the FileInfo.path suggestion quite a bit though. I didn't realize that existed. I will update the PR to use those functions.

Reply:
> This filepath is being used to update the image path for the tileset, not the tileset itself.

Makes sense, thanks for going into a bit of detail!

> I do like the FileInfo.path suggestion quite a bit though. I didn't realize that existed. I will update the PR to use those functions.

Note that in the [Files view](https://github.com/mapeditor/tiled-to-godot-export/pull/40/files) you can click "Add suggestion to batch" and eventually commit all suggestions you would apply as-is. :-)

Reply:
Hey @bjorn , I went ahead and made some updates. I was able to clean up the logic quite a bit after your suggestions.

Reply:
Yeah, that looks much better! 👍 I've approved it now since on JS or Tiled API level I have nothing to add, but I didn't test the change.